### DISCUSSION TITLE
Possability to show footer and add count or sum

### DISCUSSION THREAD

**Discussion by Thoemmy** [revolist/revogrid#487] [General]
Is there a possability to show a footer where i can add some custom information such as count or sum of a column?

I tried following:

```html
  <revo-grid
    style="height: 60vh"
    [columns]="columns"
    [source]="source"
    [pinnedBottomSource]="pinnedSource"
  >
  <div slot="data-rgCol-rowPinEnd">Custom Content</div>
</revo-grid>
```

```typescript
export class RevoGridComponent implements OnInit, OnDestroy {
  private destroy$ = new Subject<void>();

  source: any[] = [
    {
      material: 'AB-0014-4556',
      documents: 'TT-LO-45568796',
    },
    {
      material: 'AB-8877-4556',
      documents: 'TT-LO-957594',
    },
    {
      material: 'AB-7755-6788',
      documents: 'TT-LO-3466444',
    },
  ];

  columns: ColumnRegular[] = [
    {
      prop: 'material',
      name: 'Material',
      autoSize: false,
      size: 140,
      readonly: true,
      filter: true,
      pin: 'colPinStart',
      sortable: true,
      order: 'asc',
    },
    {
      prop: 'documents',
      name: 'Documents',
      readonly: true,
      autoSize: false,
      size: 160,
    }
  ];


  pinnedSource= [{}];


  ngOnDestroy(): void {
    this.destroy$.next();
  }
}

```

Is there a way how to do it right? xD thanks in advance

**Comment by m2a2x** [comment]
hello @Thoemmy 
I would just do in in a separate cell at the bottom, you could pin row if needed to the bottom

**Reply by Thoemmy** [reply]
Hello @m2a2x 
yeah i also tried it with a new row but it can be filtered which is not good.

I need this for multiple columns, i tried with slots but its not working.
I now added outside the table a div which includes the toal counts or sum and is not responsive to the column size changes.

Probably this feature would be great in future

**Reply by revolist** [reply]
Sounds like a plugin (you can put your custom elements inside of the grid with plugin)

**Reply by Jordan-Hall** [reply]
This is quiet a standard feature. I'm happy to make it but how do you hook into the filter data?
